### PR TITLE
Get vote description

### PR DESCRIPTION
**Pull request by repoman**
Added `description` property to returned vote objects.

### CODE REVIEW

File: lib/fetch_votes.js [hunk 1]
```diff
@@ -13,6 +14,19 @@ const fetchVotes = async () => {
   const votes = await voting.votes()
   console.debug(`fetchVotes`, `votes: ${votes}`)
 
+  const apps = await fetchApps()
+  const voteDescriptions = {}
+
+  // make concurrent requests for scripts
+  await Promise.all(votes.map(async vote => {
+    const describedScript = await describeScript(vote.script, apps)
```
Comment (nya-elimu) [inline lib/fetch_votes.js:22]: Why are apps required?

**Review by nya-elimu — APPROVED** [review]

**Review by nya-elimu — CHANGES_REQUESTED** [review]
I'm not able to access /votes:

```
$ yarn build
yarn run v1.22.4
$ node app.js
Server listening at http://localhost:3000
req.url: "/votes"
fetchVotes
fetchVotes voting: [object Object]
fetchVotes votes: [object Object],[object Object],[object Object],[object Object],[object Object]
fetchApps
fetchApps org: [object Object]
fetchApps apps: [object Object],[object Object],[object Object],[object Object],[object Object],[object Object],[object Object]
events.js:291
      throw er; // Unhandled 'error' event
      ^

Error: getaddrinfo ENOTFOUND mainnet.eth.aragon.network
    at GetAddrInfoReqWrap.onlookup [as oncomplete] (dns.js:66:26)
Emitted 'error' event on WebSocket instance at:
    at ClientRequest.<anonymous> (/Users/nya-elimu/GitHub/aragon-dao-webapp/node_modules/@ethersproject/providers/node_modules/ws/lib/websocket.js:555:15)
    at ClientRequest.emit (events.js:314:20)
    at TLSSocket.socketErrorListener (_http_client.js:469:9)
    at TLSSocket.emit (events.js:314:20)
    at emitErrorNT (internal/streams/destroy.js:100:8)
    at emitErrorCloseNT (internal/streams/destroy.js:68:3)
    at processTicksAndRejections (internal/process/task_queues.js:80:21) {
  errno: -3008,
  code: 'ENOTFOUND',
  syscall: 'getaddrinfo',
  hostname: 'mainnet.eth.aragon.network'
}
error Command failed with exit code 1.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
```

**Review by nya-elimu — APPROVED** [review]
The `Error: getaddrinfo ENOTFOUND mainnet.eth.aragon.network` seems to be a temporary issue.

Ready to be merged!

File: lib/fetch_votes.js (hunk 1 above)
Comment (repoman) [reply lib/fetch_votes.js:22]: I’m not quite sure, but it seems to be required by the Radspec interpreter.

**Pull request merged**